tests/requirements.txt

```diff
@@ -9,3 +9,4 @@ pytest-html
 flake8
 koji
 requests-mock
+git+https://github.com/containerbuildsystem/osbs-client
```

Comment:
This will have to be in the top-level requirements.txt (not just for tests), but I guess it's fine to leave that for the Cachito changes